README.md

```diff
@@ -23,832 +23,10 @@ This extension is for REST code generation for client side only. If you're looki
 
 ## Getting Started
 
-Add the following dependency to your project's `pom.xml` file:
+You can learn more in [Quarkus Openapi Generator Documentation](http://docs.quarkiverse.io/quarkus-openapi-generator/dev/index.html).
```

Comment:
@hbelmiro, I added this link. I think that will be the future link based in outer extensions that I saw. But, let me know if I need change.